### ISSUE TITLE
insertText() and popped out window

### ISSUE THREAD

**Issue by crsh** [rstudio/rstudioapi#55]
Hi,

I'm trying to use `insertText()` with a popped out window.

Let's say I have a source pane with context id `3B337525` and a popped out source window with id `2DD041B0`. When I run `rstudioapi::insertText(text = "test", id = "2DD041B0")` in the popped out source window, `test` is inserted into the document. When I run the same code in the source pane or the console it appears that nothing happens. Conversely, when I run `rstudioapi::insertText(text = "test", id = "3B337525")` in the console or source pane `text` is inserted into the pane document, but if I run the same code in the popped out window, again, nothing seems to happen.

My use case is an addin that cannot insert text in a popped out window but in source panes. I'm trying this with the current development snapshot from this repository. What am I doing wrong?

```{r}
> devtools::session_info()
Session info ---------------------------------------------------------------------------------------------------
 setting  value                       
 version  R version 3.4.1 (2017-06-30)
 system   x86_64, linux-gnu           
 ui       RStudio (1.0.153)           
 language de_DE                       
 collate  de_DE.UTF-8                 
 tz       Europe/Berlin               
 date     2017-08-23                  

Packages -------------------------------------------------------------------------------------------------------
 package    * version    date       source              
 assertthat   0.2.0      2017-04-11 cran (@0.2.0)       
 base       * 3.4.1      2017-07-08 local               
 citr       * 0.2.0.9047 2017-08-23 local (crsh/citr@NA)
 compiler     3.4.1      2017-07-08 local               
 datasets   * 3.4.1      2017-07-08 local               
 devtools     1.13.2     2017-06-02 CRAN (R 3.4.0)      
 digest       0.6.12     2017-01-27 CRAN (R 3.4.0)      
 graphics   * 3.4.1      2017-07-08 local               
 grDevices  * 3.4.1      2017-07-08 local               
 htmltools    0.3.6      2017-04-28 cran (@0.3.6)       
 httpuv       1.3.5      2017-07-04 cran (@1.3.5)       
 memoise      1.1.0      2017-04-21 CRAN (R 3.4.0)      
 methods    * 3.4.1      2017-07-08 local               
 mime         0.5        2016-07-07 CRAN (R 3.4.0)      
 miniUI       0.1.1      2016-01-15 cran (@0.1.1)       
 R6           2.2.2      2017-06-17 CRAN (R 3.4.0)      
 Rcpp         0.12.12    2017-07-15 cran (@0.12.12)     
 shiny        1.0.4      2017-08-14 cran (@1.0.4)       
 stats      * 3.4.1      2017-07-08 local               
 tools        3.4.1      2017-07-08 local               
 utils      * 3.4.1      2017-07-08 local               
 withr        1.0.2      2016-06-20 CRAN (R 3.4.0)      
 xtable       1.8-2      2016-02-05 CRAN (R 3.4.0)  
```

**Comment by kevinushey** [comment]
I don't think you're doing anything wrong here -- it looks like this may be a bug in RStudio's handling of text insertion in popped out documents.

If I can summarize what you're seeing here, effectively, attempting to insert text in a window different from the currently active window seems to fail. Does that sound right? That is, this 'cross-window' text insertion seems to be failing?

**Comment by crsh** [comment]
Exactly. It works across panes (and console) in the same window but not across windows.

Sent from my phone

**Comment by kevinushey** [comment]
Got it! This is indeed an issue in the RStudio IDE; this will be resolved in the v1.1 release (or, you should see it in the next few days as part of the [preview release](https://www.rstudio.com/products/rstudio/download/preview/))

**Comment by crsh** [comment]
I've tried with the latest stable release and it seems this issue has been resolved. Thanks!
